common/src/theme.rs

```diff
@@ -0,0 +1,30 @@
+use serde::{Deserialize, Serialize};
+use std::{fs, io};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub enum ThemeType {
+    Board,
+    Piece,
+}
+
+pub async fn get_themes(theme_type: ThemeType) -> io::Result<Vec<String>> {
+    let mut themes = Vec::new();
+    let dir_path = match theme_type {
+        ThemeType::Board => "images/boards/",
+        ThemeType::Piece => "images/pieces/",
+    };
+
+    for entry in fs::read_dir(dir_path)? {
```

Comment:
Looks like there is an `fs` in `async_std`. Maybe if this is an async function anyway we could use that?